src/components/Login/PhoneForm/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { SubmitHandler } from 'react-hook-form'
+
+import FormFields, { FormValues } from '@/components/Login/PhoneForm/formfields'
+import Form from '@/components/UI/FormComponents/Form'
+
+const PhoneForm = () => {
+  const handleSubmit: SubmitHandler<FormValues> = (data) => {
+    console.log(data)
```

Comment:
This should throw a lint error? Maybe put in a TODO

Reply:
We didn't actually need a submit function for the phone login, so I've changed it so submit function is optional now and removed it.